mermithid/processors/misc/KrComplexLineShape.py

```diff
@@ -54,7 +54,7 @@ def InternalConfigure(self, params):
         self.fix_scatter_proportion = reader.read_param(params, 'fix_scatter_proportion', True)
         if self.fix_scatter_proportion == True:
             self.scatter_proportion = reader.read_param(params, 'gas1_scatter_proportion', 0.8)
-            logger.info('Using an H2 scatter proportion of {} with gases {}'.format(self.gases, self.scatter_proportion))
+            logger.info('Using an H2 scatter proportion of {} with gases {}'.format(self.scatter_proportion, self.gases))
```

Comment:
why do we have a KrComplexLineshape processor and a MultiGasComplexLineshape processor? This processor is also capable of doing multi gases it seems. Do we need both processors? What are the differences between them. Please add detailed description of processors, their capabilities and the differences to the other lineshape processor.